packages/server/src/api/routes/public/middleware/testErrorHandling.ts

```diff
@@ -0,0 +1,24 @@
+import { Ctx } from "@budibase/types"
+import environment from "../../../../environment"
+
+export default async (ctx: Ctx, next: any) => {
+  try {
+    await next()
+  } catch (err: any) {
+    if (
+      !(environment.isTest() && ctx.headers["x-budibase-include-stacktrace"])
```

Comment:
Could we throw an error if this is used outside `environment.isTest()` at register time instead of runtime?

Reply:
Good idea! Done.